usaspending_api/awards/management/commands/helpers.py

```diff
@@ -0,0 +1,108 @@
+from datetime import datetime
+
+from django.db.models import Q
+
+from usaspending_api.awards.models import Award
+from usaspending_api.references.models import Agency, Location, RefCountryCode
+
+
+def up2colon(input_string):
+    'Takes the part of a string before `:`, if any.'
```

Comment:
This throws an error if `input_string` is None, might want to check that and just return None if input_string is none

I'm not exactly sure how/when/why the input is ever None, I think it has something to do with csvreader, but while testing this did come up.